### PR TITLE
Enable style_transfer model using New API

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Type of Change

feature
API not changed

## Description

detail description 
JIRA ticket: ILITV-2624

## Expected Behavior & Potential Risk

style_transfer can quantize and benchmark well using new api.

## How has this PR been tested?

Pre-CI and extension test.

## Dependency Change?

No.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -301,7 +301,7 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>*Arbitrary Style Transfer</td>
```
Comment (chensuyue) [inline examples/README.md:301]: Please also remove the `*` after migrate into 2.0 API.

**Comment by lvliang-intel** [comment]
extension test:
https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-extension-TFnewAPI/339/artifact/report.html

File: examples/README.md [hunk 2]
```diff
@@ -298,10 +298,10 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td><a href="./tensorflow/recommendation/wide_deep_large_ds/quantization/ptq">pb</a></td>
   </tr>
   <tr>
-    <td>*Arbitrary Style Transfer</td>
+    <td>Arbitrary Style Transfer</td>
     <td>Style Transfer</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/style_transfer/arbitrary_style_transfer/quantization/ptq">ckpt</a></td>
+    <td><a href="./examples/tensorflow/style_transfer/arbitrary_style_transfer/quantization/ptq">ckpt</a></td>
```
Comment (chensuyue) [inline examples/README.md:304]: link need to fix.

File: examples/tensorflow/oob_models/quantization/ptq/README.md [hunk 3]
```diff
@@ -16,8 +18,8 @@ This document is used to list steps of reproducing Intel Optimized TensorFlow OO
   ```
 > Note: Supported Tensorflow [Version](../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/oob_models/quantization/ptq/README.md:19]: link need to fix.

File: examples/tensorflow/recommendation/wide_deep_large_ds/quantization/ptq/README.md [hunk 4]
```diff
@@ -4,20 +4,22 @@ Step-by-Step
...
 ```shell
 # Install Intel® Neural Compressor
 pip install neural-compressor
 ```
-### 2. Install Intel Tensorflow
+### Install Intel Tensorflow
 ```shell
 pip install intel-tensorflow
 ```
 > Note: Supported Tensorflow [Version](../../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/recommendation/wide_deep_large_ds/quantization/ptq/README.md:20]: link need to fix.

File: examples/tensorflow/semantic_image_segmentation/3dunet-mlperf/quantization/ptq/README.md [hunk 5]
```diff
@@ -4,21 +4,23 @@ Step-by-Step
...
 # Install Intel® Neural Compressor
 pip install neural-compressor
 ```
 
-### 2. Install Intel Tensorflow
+### Install Intel Tensorflow
 ```shell
 pip install intel-tensorflow
 ```
 > Note: Supported Tensorflow [Version](../../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/semantic_image_segmentation/3dunet-mlperf/quantization/ptq/README.md:21]: link need to fix.

File: examples/tensorflow/style_transfer/arbitrary_style_transfer/quantization/ptq/README.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
+> Note: Supported Tensorflow [Version](../../../../../../README.md#supported-frameworks).
```
Comment (chensuyue) [inline examples/tensorflow/style_transfer/arbitrary_style_transfer/quantization/ptq/README.md:20]: link need to fix.

**Review by Spycsh — APPROVED** [review]

**Pull request merged**